source/sass/views/home.scss

```diff
@@ -206,6 +206,10 @@
       font-size: 18px;
       font-weight: 700;
     }
+
+    &:first-child {
```

Comment:
Ideally we should stop adding new SCSS code and just rely on Tailwind as much as we can. However, since this Homepage section is still using our existing Bootstrap/SCSS classes, I'm okay with this being an exception of "no new SCSS".

Reply:
This will most likely get refactored in the remaining Homepage Refresh tickets. We shall be going full tailwind by then.